graphdatascience/query_runner/neo4j_query_runner.py

```diff
@@ -248,17 +248,20 @@ def _log(self, job_id: str, future: "Future[Any]", database: Optional[str] = Non
                     continue
 
             progress_percent = progress["progress"][0]
-            if not progress_percent == "n/a":
-                task_name = progress["taskName"][0].split("|--")[-1][1:]
-                pbar = pbar or tqdm(total=100, unit="%", desc=task_name)
-            else:
+            if progress_percent == "n/a":
                 return
 
+            task_name = progress["taskName"][0].split("|--")[-1][1:]
```

Comment:
I dont understand why we only take the first taskName and not update all related bars